README.md

```diff
@@ -170,6 +170,34 @@ To do this, add the following to your `.vscode/settings.json` file:
 },
 ```
 
+## Zed Setup
+
+Because this repository uses a custom sysroot with custom libraries and binaries, you need to set up your environment accordingly.
+Here is a suggested configuration for Zed that you can put in ".zed/settings.json":
+```json
+{
```

Comment:
Not strictly related to LSP config, but could we please also fix the preferred line length (to align with rustfmt)?

```suggestion
{
   "preferred_line_length": 100,
```

Reply:
I am going to change this to format on save using rust so this shouldn't be needed.  I'm not a fan of the behavior when typing longer lines.  Better to format on save.

Reply:
Added the language server formatting on save.

Reply:
> I am going to change this to format on save using rust so this shouldn't be needed. I'm not a fan of the behavior when typing longer lines.

What behaviour? This doesn't wrap lines when you type them, except if you turn `soft_wrap` on. I don't expect people to have it on, and if they do, then we want to wrap at 100 char and not 80. My use case is when I use the Vim binding to rewrap a comment, I want it to wrap on 100 char.

> Better to format on save.

Comments aren't reformatted by autoformat on save.

Also, I'm pretty sure formatting on save is already on by default?